identity-server/src/IdentityServer/Services/Default/DefaultJwtRequestUriHttpClient.cs

```diff
@@ -56,19 +53,19 @@ public async Task<string> GetJwtAsync(string url, Client client)
                 if (!string.Equals(response.Content.Headers.ContentType.MediaType,
                         $"application/{JwtClaimTypes.JwtTypes.AuthorizationRequest}", StringComparison.Ordinal))
                 {
-                    _logger.LogError("Invalid content type {type} from jwt url {url}",
-                        response.Content.Headers.ContentType.MediaType, url);
+                    _sanitizedLogger.LogError("Invalid content type {type} from jwt url {url}",
+                        response.Content.Headers.ContentType.MediaType, url.ReplaceLineEndings(string.Empty));
```

Comment:
The call to `ReplaceLineEndings` is unnecessary here, since all log arguments are sanitized using `ILoggerExtensions.SanitizeLogParameter`, which already calls `ReplaceLineEndings()` internally.